lib/plugins/aws/package/lib/stripNullPropsFromTemplateResources.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  stripNullPropsFromTemplateResources() {
+    const resources = this.serverless.service.provider.compiledCloudFormationTemplate.Resources;
+
+    if (Object.keys(resources).length > 0) {
```

Comment:
Do we need that check? I believe if we simply skip it, it will behave in the same way as the `for .. of` loop will only have an empty array to iterate over.